node/nodeRunner.go

```diff
@@ -476,26 +476,19 @@ func (nr *nodeRunner) createApiFacade(
 
 	ef.SetSyncer(currentNode.coreComponents.SyncTimer())
 
-	err = upgradableHttpServer.GetHttpServer().Close()
-	if err != nil {
-		return nil, err
-	}
-
 	err = upgradableHttpServer.UpdateFacade(ef)
 	if err != nil {
 		return nil, err
 	}
 
-	go upgradableHttpServer.GetHttpServer().Start()
```

Comment:
we should remove upgradeable http server implementation and also simplify the structs because we do no longer need to restart it.

Reply:
refactored